framework/set/provisioning/providers/harvester/harvesterMachineConfig.go

```diff
@@ -0,0 +1,31 @@
+package harvester
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/rancher/tfp-automation/defaults/resourceblocks/nodeproviders/harvester"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// SetHarvesterRKE2K3SMachineConfig is a helper function that will set the Harvester RKE2/K3S
+// Terraform machine configurations in the main.tf file.
```

Comment:
Can you put this comment on the same line as line 11? Reads a bit weird having it split like this.